src/app/give-help/accommodation-form/accommodation-form.component.ts

```diff
@@ -3,6 +3,16 @@ import { defaults } from '@app/shared/utils';
 import { AccommodationOfferDefinitionDTO } from '../../core/api/model/accommodationOfferDefinitionDTO';
 import { PREFIXES, LANGUAGES, LENGTHOFSTAY } from '@app/shared/consts';
 import { AccommodationsResourceService } from '@app/core/api';
+import { CorePath } from '@app/shared/models/core-path.model';
+import { SnackbarService } from '@app/shared/services/snackbar.service';
+import { ALERT_TYPES } from '@app/shared/models/';
+import { take } from 'rxjs/operators';
+import { Router } from '@angular/router';
+// Waiting for TransportOfferDefinitionDTO receive a phoneNumber prop
+interface AccommodationOfferDefinition extends AccommodationOfferDefinitionDTO {
+  phoneNumber?: string;
+}
```

Comment:
Can you report that on the Backend Teams channel?

Reply:
Zgłoszone ponad tydzień temu, jest na to ticket https://jira.sysopspolska.pl/browse/POM-188?jql=text%20~%20%22phonenumber%22